### PR TITLE
new(blog): modern eBPF probe in Falco 0.35

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind content

**Any specific area of the project related to this PR?**

/area blog

**What this PR does / why we need it**:

This PR adds a new blog post for the modern eBPF probe enhancements in Falco 0.35

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/falco-modern-bpf-0-35-0/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Conversely, reducing the buffer size can help enhance system speed, but it may also result in an increased number of syscall drops. It's crucial to exercise caution when experimenting with this configuration option, taking into consideration the trade-off between performance and syscall drops ⚠️
+
+While the ability to adjust buffer sizes is a cool feature, it is available to all drivers. So, what sets the modern probe apart? Well, with this new driver, you have the added capability to manipulate the number of buffers. This means that the traditional rule of having one buffer per CPU is no longer a strict requirement. Unlike the old drivers, where the only possible configuration was one buffer per CPU, the modern probe introduces flexibility in this aspect, opening up new possibilities and alternative scenarios.
+
+As an illustration, one such scenario is the allocation of a ring buffer for every 3 CPUs.
+
+![](/blog/falco-modern-bpf-0-35-0/images/falco-modern-bpf-03.png)
```
Comment (vjjmiras) [inline content/en/blog/falco-modern-bpf-0-35-0/index.md:77]:
```suggestion
![Buffers allocation using the modern eBPF probe](/blog/falco-modern-bpf-0-35-0/images/falco-modern-bpf-03.png)
```

File: content/en/blog/falco-modern-bpf-0-35-0/index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    ```plain
+    tracing program type is not supported
+    ```
+
+## Buffers allocation
+
+Every Falco driver utilizes shared buffers to facilitate the transmission of security events between the kernel and the userspace. To be more specific, there is an individual buffer allocated for each online CPU, ensuring efficient handling of events.
+
+![](/blog/falco-modern-bpf-0-35-0/images/falco-modern-bpf-02.png)
```
Comment (vjjmiras) [inline content/en/blog/falco-modern-bpf-0-35-0/index.md:65]:
```suggestion
![Buffers allocation using the current eBPF probe](/blog/falco-modern-bpf-0-35-0/images/falco-modern-bpf-02.png)
```

File: content/en/blog/falco-modern-bpf-0-35-0/index.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+However, feel free to experiment and find the configuration that best suits your system. Just remember the following guideline: having more buffers can reduce the likelihood of drops but will increase the overall memory footprint. On the other hand, reducing the number of buffers can help decrease memory consumption but may lead to an increased risk of drops.
+
+## Least privileged mode
+
+Similar to the current probe, the modern probe can operate in _least privileged_ mode. However, to ensure proper functionality, Falco always mandates a minimum of two capabilities: `CAP_SYS_RESOURCE` and `CAP_SYS_PTRACE`. Additional required capabilities vary depending on your specific kernel version, like the `CAP_SYS_ADMIN` capability for older kernels, which can be replaced by the `CAP_PERFMON` and `CAP_BPF` ones when running on a kernel newer than 5.8.
+
+![](/blog/falco-modern-bpf-0-35-0/images/falco-modern-bpf-04.png)
```
Comment (vjjmiras) [inline content/en/blog/falco-modern-bpf-0-35-0/index.md:93]:
```suggestion
![Newer kernels allow more granularity when using Linux capabilities](/blog/falco-modern-bpf-0-35-0/images/falco-modern-bpf-04.png)
```

**Comment by Andreagit97** [comment]
Thank you @vjjmiras for the suggestions!

**Review by Issif — APPROVED** [review]

**Review by vjjmiras — APPROVED** [review]

**Pull request merged**